spec/forms/teacher_interface/alternative_name_form_spec.rb

```diff
@@ -1,46 +1,92 @@
 require "rails_helper"
 
 RSpec.describe TeacherInterface::AlternativeNameForm, type: :model do
-  describe "validations" do
-    it { is_expected.to validate_presence_of(:application_form) }
-  end
-
   let(:application_form) { build(:application_form) }
 
-  describe "#valid?" do
-    subject(:valid?) { form.valid? }
-
-    let(:form) do
+  describe "validations" do
+    subject(:form) do
       described_class.new(
         application_form:,
         has_alternative_name:,
         alternative_given_names:,
         alternative_family_name:,
       )
-    end
```

Comment:
I think we need to keep this `end` as otherwise the `subject` block doesn't end correctly, it should only contain the `described_class.new`.